docs/api-docs/apps/guide/apps-02-types.md

```diff
@@ -3,88 +3,99 @@
 <div class="otp" id="no-index">
 
 ### On This Page

```

Comment:
The section headings on this page should be [sentence case](https://intranet.bigcommerce.com/display/SST/Headings+and+Titles)